services/cell-logger/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# BUILDER
+FROM python:3 as builder
+
+WORKDIR /usr/src/app
+
+# Download latest listing of available packages
+RUN apt-get -y update
+
+# Needed for building dbus-python
+RUN apt-get install -y --no-install-recommends python-pip pkg-config libdbus-1-dev libglib2.0-dev libdbus-glib-1-dev python3-dbus
+
+# Activate virtualenv
+RUN python -m venv /opt/venv
+
+# Make sure we use the virtualenv
+ENV PATH="/opt/venv/bin:$PATH"
```

Comment:
Seems like it might be time to factor this out into a base container. Maybe I can make that a todo for me after the PR is merged.